EduVPN-redesign/AppDelegate.swift

```diff
@@ -167,6 +169,43 @@ extension AppDelegate {
         }
     }
 
+    @IBAction func importOpenVPNConfig(_ sender: Any) {
+        guard let mainWindow = mainWindow else { return }
+        guard let persistenceService = environment?.persistenceService else { return }
+
+        mainWindow.makeKeyAndOrderFront(nil)
+        NSApp.activate(ignoringOtherApps: true)
+
+        let openPanel = NSOpenPanel()
+        openPanel.prompt = NSLocalizedString("Import", comment: "")
+        openPanel.allowedFileTypes = ["ovpn"]
+        openPanel.allowsMultipleSelection = false
+        openPanel.beginSheetModal(for: mainWindow) { response in
+            guard response == .OK else { return }
+            guard let url = openPanel.urls.first else { return }
+            var importError: Error?
+            do {
+                let instance = try OpenVPNConfigImportHelper.copyConfig(from: url)
+                persistenceService.addOpenVPNConfiguration(instance)
+            } catch {
+                importError = error
+            }
+
+            self.mainViewController?.refresh()
+
+            if let importError = importError {
+                if let navigationController = self.environment?.navigationController {
+                    navigationController.showAlert(for: importError)
+                }
+            } else {
+                let alert = NSAlert()
+                alert.messageText = NSLocalizedString("OpenVPN config imported", comment: "")
+                alert.informativeText = ""
+                alert.beginSheetModal(for: mainWindow)
```

Comment:
Seems unneeded?

Reply:
I assume you're saying the alert is not required. It's because the user can be in any screen when importing (can import even when a tunnel is active), so the alert exists to provide visual feedback that the importing succeeded.

Reply:
Yes, I think the user would expect that it would show up in the list as that is the only logical place for it. Especially if the user is already on that screen, the alert is just in the way. If the user is on another screen, this alert doesn't really explain that it now is in the list.

Reply:
Okay, that makes sense. Removed the alert in commit 33bb4a5.